content/guides/guides/debugging.md

```diff
@@ -66,7 +66,7 @@ execution and add a `debugger` at the appropriate time:
 ```js
 cy.visit('/my/page/path')
 
-cy.get('.selector-in-question').then(($selectedElement) => {
+cy.get('[data-testid=selector-in-question]').then(($selectedElement) => {
```

Comment:
```suggestion
cy.get('[data-testid="selector-in-question"]').then(($selectedElement) => {
```

I think the attribute values need quotes around them like [this example](https://docs.cypress.io/api/commands/get#Find-5-elements-with-the-given-data-attribute).

Reply:
Technically, I think that's more correct. I used the example that we show at the bottom of the table here as a guide https://docs.cypress.io/guides/references/best-practices#How-It-Works and confirmed this works. 

I was going for consistency with these edits but totally open to making a larger scale find and replace if we decide we want something else.

Reply:
Or...maybe we should update that table to be more technically correct and the examples follow that.

Reply:
@jaffrepaul my vote would be to update the table to avoid similar questions in the future.

Reply:
I can dig that. I'll need to make another pass and update some other selectors. Easy find and replace tho.